lib/handlers/marshalling/pool-marshaller.ts

```diff
@@ -1,6 +1,6 @@
 import { Pool } from '@uniswap/v3-sdk'
 import { FeeAmount } from '@uniswap/v3-sdk/dist/constants'
-import { MarshalledToken, TokenMarshaller } from './token-marshaller'
+import { MarshalledToken, TokenMarshaller } from '../router-entities/route-caching/marshalling/token-marshaller'
```

Comment:
it seems like this file was moved to a different place, let's move all of the marshallers, or none of them it feels confusing to have them separated.